### PR TITLE
Lesson 02

### PR DESCRIPTION
**Pull request by SafonovVladimir**
Add all project files to main

### CODE REVIEW

File: lesson_02/job1/main.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    message = save_sales_to_local_disk(date=date, raw_dir=raw_dir)
+
+    return {
+        "message": f"{message}",
+    }, 201
+
+
+if __name__ == "__main__":
+    app.run(host="localhost", port=8081)
```
Comment (tmikhailovska) [inline lesson_02/job1/main.py:46]: could be moved to settings

File: lesson_02/job2/main.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    create_avro_files(raw_dir, stg_dir)
+
+    return {
+        "message": "Data save successfully to local storage",
+    }, 201
+
+
+if __name__ == "__main__":
+    app.run(host="localhost", port=8082)
```
Comment (tmikhailovska) [inline lesson_02/job2/main.py:40]: could be moved to settings

**Review by tmikhailovska — APPROVED** [review]
Looks good, only host and port could be moved to settings

File: lesson_02/job1/main.py (hunk 1 above)
Comment (SafonovVladimir) [reply lesson_02/job1/main.py:46]: Yep, you are rigth. I could be better

File: lesson_02/job2/main.py (hunk 2 above)
Comment (SafonovVladimir) [reply lesson_02/job2/main.py:40]: May I merge that PR or it's better to wait Denys approve?

Comment (DenysKulemza) [reply lesson_02/job2/main.py:40]: @SafonovVladimir, you can merge. If it's approved by the methodist then it can be merged

**Pull request merged**